projects/storefrontapp-e2e-cypress/cypress.config.ci.ts

```diff
@@ -20,7 +20,8 @@ export default defineConfig({
   },
   videoUploadOnPasses: false,
   env: {
-    API_URL: 'https://20.83.184.244:9002',
+    // API_URL: 'https://20.83.184.244:9002',
+    API_URL: 'https://spartacus-dev-asx1.eastus.cloudapp.azure.com:8443',
```

Comment:
`major:` let's not forget to revert this change

Reply:
reverted